var/spack/repos/builtin/packages/py-us/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyUs(PythonPackage):
+    """US state meta information and other fun stuff."""
```

Comment:
I've actually used this Python library before for my research, thanks for adding it!

Reply:
Awesome.